doc/whatsnew/fragments/4836.newcheck

```diff
@@ -0,0 +1,3 @@
+Add a ``shadowed-import`` message for aliased imports.
```

Comment:
The file name should probably be `new_check`

Reply:
Thanks! This will definitely help. It has been an issue for me a couple of times as well.